cmd/clairctl/docker/push.go

```diff
@@ -0,0 +1,68 @@
+package docker
```

Comment:
My understanding is that we need to do `clairctl push` and then `clairctl analyse`, right? Why don't we just have `clairctl analyze` - which would make Clair analyze the layers first if necessary (basically if Clair says that the layer doesn't exist).